deploy/helms/legion/templates/edi/edi-server.yaml

```diff
@@ -54,6 +54,18 @@ spec:
         - containerPort: 80
           name: http
           protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /
+            port: 80
+          initialDelaySeconds: 120
+          periodSeconds: 3
+        readinessProbe:
```

Comment:
Do we really need liveness and readiness probe in this case?

Reply:
addressed